python/afdko/otfautohint/hintstate.py

```diff
@@ -369,23 +433,29 @@ def remExtraBends(self):
             for hsd in self.decreasingSegs[lo:hi]:
                 assert hsd.loc == hsi.loc
                 if hsd.min < hsi.max and hsd.max > hsi.min:
-                    if (hsi.type == hintSegment.sType.BEND and
-                            hsd.type != hintSegment.sType.BEND and
-                            hsd.type != hintSegment.sType.GHOST and
-                            (hsd.max - hsd.min) > (hsi.max - hsi.min) * 3):
+                    if (
+                        hsi.type == hintSegment.sType.BEND
+                        and hsd.type != hintSegment.sType.BEND
+                        and hsd.type != hintSegment.sType.GHOST
+                        and (hsd.max - hsd.min) > (hsi.max - hsi.min) * 3
+                    ):
                         hsi.deleted = True
-                        log.debug("rem seg loc %g from %g to %g" %
-                                  (hsi.loc, hsi.min, hsi.max))
+                        log.debug(
+                            "rem seg loc %g from %g to %g" % (hsi.loc, hsi.min, hsi.max)
```

Comment:
Surprised there aren't flake8 complaints about these long lines.

We're not going to go through and insist that every otherwise-unchanged line is formatted back to what it was, but let's revert the ones around here for clarity and flake8 purposes. Best to keep this PR mostly tracking what needed to be changed for these type changes.